internal/model/naming.go

```
@@ -61,8 +61,42 @@ func ColumnName(pluginColumn *plugin.Column, pos int) string {
 }
 
+// sanitizePyIdentifier maps every rune that cannot appear in a Python
+// identifier to an underscore. Returns "" when nothing identifier-worthy
+// remains; digitPrefix (plus an underscore) is prepended when the result
+// starts with a digit - a bare underscore prefix would break attrs and
+// pydantic, which treat leading-underscore fields specially.
+func sanitizePyIdentifier(name, digitPrefix string) string {
+	var builder strings.Builder
+	for _, r := range name {
+		if unicode.IsLetter(r) || unicode.IsDigit(r) || r == '_' {
+			builder.WriteRune(r)
```

golangci-lint (2.12.2)

[error] Error return value of `builder.WriteRune` is not checked

(errcheck)